src/Psalm/Internal/Analyzer/CommentAnalyzer.php

```diff
@@ -1038,7 +1038,14 @@ public static function extractClassLikeDocblockInfo(
 
 
                     if ($method_tree_child->children) {
-                        $param_type = Type::getTypeFromTree($method_tree_child->children[0], $codebase);
+                        try {
+                            $param_type = Type::getTypeFromTree($method_tree_child->children[0], $codebase);
+                        } catch (TypeParseTreeException $e) {
+                            $msg = 'Badly-formatted @method parameters for ' . $method_entry;
+                            $msg .= '; have you checked if a variable passed by reference has a space between';
+                            $msg .= ' the ampersand and the start of the variable?';
```

Comment:
I would prefer the message to be more actionable. Like 'There should be no space between & and the variable name' or something similar.